tests/template_test.go

```diff
@@ -0,0 +1,91 @@
+package tests_test
+
```

Comment:
I think the file name is confusing, as it is not testing template functionality, but the existing templates,
i suggest to rename to something like template_example_tests

Reply:
@nellyc In CNV we want to be sure we can use Template mechanism to create fully fledged 'VirtualMachine` object that passes a consequent set of `VirtualMachine` tests. Since @gouyang also verified if the Template parameters were propagated correctly into the `VirtualMachine` object properties. This alone should be sufficient to verify the template functionality from the CNV perspective (although I would use Cirros image here rather than Fedora to speed up the test a bit). When we already know that Template mechanism is functional, we can be applied the same set of tests to all our example templates, to test the templates. Since the test aimed to test both the Template functionality (from the CNV perspective) and the example templates, I would not see it confusing.